docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -1,7 +1,186 @@
 # What does `security-status` do?
 
+## Command output
+
 The `security-status` command is used to get an overview of the packages
-installed on your machine.
+installed on your machine, and an overview of the security coverage for those
+packages.
+
```

Comment:
It always renders a bit strangely to have two headings with no text between them. I'd suggest moving the "## Command output" on line 3 to here